src/packages/infiniteloading/infiniteloading.tsx

```
@@ -77,6 +79,16 @@ export const InfiniteLoading: FunctionComponent<
 
   const classes = classNames(classPrefix, className, `${classPrefix}-${type}`)
 
+  useEffect(() => {
+    if (defaultScrollTop) {
+      const childHeight = (getRefreshTop().firstElementChild as HTMLElement).offsetHeight || 0;
```

ESLint

[error] Replace `·(getRefreshTop().firstElementChild·as·HTMLElement).offsetHeight·||·0;` with `⏎········(getRefreshTop().firstElementChild·as·HTMLElement).offsetHeight·||·0`

(prettier/prettier)